README.md

```diff
@@ -1,2 +1,6 @@
 # code_style
 Code styles, and tools for different languages
+
+# General conventions
+
+- Mobile Clients will send a header `bv-app-version` with the app version for each api request.
```

Comment:
¿What happens if we have one or more mobile application for each platform. For example an application for customer and another application for admin?
I believe the `bv-app-version` should contain `platform/typeAppr/version`; 
`platform`: maybe `android` or `iOS`
`typeApp`: i.e `Customer` or `admin`
`version`: following semver standars (`x.y.z`)